internal/builtinfunctions/functions.go

```diff
@@ -212,7 +220,8 @@ func getStringToFloatFunction() schema.CallableFunction {
 		schema.NewDisplayValue(
 			schema.PointerTo("stringToFloat"),
 			schema.PointerTo(
-				"Converts the input string to a 64-bit floating-point number\n\n"+
+				"Returns a floating point number by interpreting the the input\n"+
```

Comment:
"input string" is better than "inputted" (!) but I think "string parameter/argument/value" would be better.